.gitignore

```diff
@@ -4,3 +4,4 @@
 .kitchen/
 .kitchen.local.yml
 Gemfile.lock
+vendor/bundle/
```

Comment:
Does this resolve the problem we were seeing earlier today where the new vendor gem changes where not showing up?